src/zulip/commands.rs

```diff
@@ -173,6 +176,16 @@ pub struct PingGoalsArgs {
     pub next_update: String,
 }
 
+#[derive(clap::Parser, Debug, PartialEq, Clone)]
+pub struct BackportArgs {
+    /// Release channel this backport is pointing to. Allowed: "beta" or "stable".
+    pub channel: String,
```

Comment:
Could you please use an `enum` here for the channel and the verb, and either generate the parser with `#[derive(clap::ValueEnum)]`, or write it manually with `FromStr`? That way we won't have to check for invalid data in the backport handler, and we'll get a nice error message and a hint (in case of misspelling) for free.